app/coordinates/coordinate-model.js

```diff
@@ -0,0 +1,13 @@
+var pool = require('../database');
+
+async function selectAllCoordinates(){
```

Comment:
Also use arrow functions instead of the function keyword, e.g:

const selectAllCoordinates = async () => {
  // code here
};